Source/render/GroupRenderer.swift

```diff
@@ -22,18 +26,14 @@ class GroupRenderer: NodeRenderer {
     override func doAddObservers() {
         super.doAddObservers()
 
-        guard let group = group else {
-            return
-        }
-
         group.contentsVar.onChange { [weak self] _ in
             self?.updateRenderers()
         }
         observe(group.contentsVar)
-    }
 
-    override func node() -> Node? {
-        return group
+        group.placeVar.onChange { [weak self] (_) in
```

Comment:
1. You didn't clean up children `absPlace` when group place updated. You need to override `freeCachedAbsPlace` and clean children as well. 
2. Such an implementation adds two handlers for group. One here and another in `super. doAddObservers`. This one can be removed.